Sources/ViewController.swift

```diff
@@ -107,4 +109,12 @@ class ViewController: UIViewController {
         )
         return labelImage
     }
+
+    private func iconWithRoundedRectangle(name: String) -> UIView {
+        let icon = AFIcon.named(name, scheme: .grayscale, size: CGSize(width: 50, height: 50))
+        let image = try? UIImage(color: .yellow, size: CGSize(width: 200, height: 92), radius: 50, isDashed: true)
+
+        let newImage = try? image!.union(icon)
```

Comment:
Don't force unwrap here, it's possible to crash (though unlikely).

Reply:
Curious what the correct implementation would be then? Technicallyyyyyy, we force unwrap when we try to put it into the labeled image. 

In Affirm production, I'm doing `modifiedImage ?? originalImage`. Does that seem like the correct way to fail?

Reply:
https://github.com/E-B-Smith/Colors/pull/5/commits/99e51bffade1d9ff7514f7d03e902cf5ac834524

Reply:
That seems like a good way to fail. 

In practice I use a default dummy image as a place holder that I expect will always work unless something is really messed up. Like:
```
    private func iconWithRoundedRectangle(name: String) -> UIView {
        var image = UIImage(named: "cry")! // I expect this to always be available, for instance.
        if let newImage = try?
            UIImage(color: .red, size: CGSize(width: 200, height: 92), radius: 20, isDashed: true)
                .union(AFIcon.named(name, scheme: .grayscale, size: CGSize(width: 50, height: 50))) {
            image = newImage
        }
        return LabeledImage(text: "Icon with oval background", image: image)
    }
```